DEPLOYMENT.md

```diff
@@ -35,19 +56,182 @@ Our Docker images our tagged in four possible ways:
 Typically, we use *unstable* for testing and development and *latest* for production.
 We recommend you follow this if you are unsure which to use.
 
+## Installing Dependencies
```

Comment:
As discussed, if we're going to be this explicit, we might want to just provide a script that can be copied, with good comments? I think @JRegimbal had reservations about this approach in the past, but worth discussing?

Reply:
Did we decide not to make a sample script with comments for this? Which could also include a sample output file so people can see what it *should* look like when it works? Or even just have a transcript of it working with manually run commands if we don't want to maintain a script? If we aren't doing that, and we do it in the copy/paste style, with command clearly delineated and with a copy/paste control? (I haven't seen this style of command explanation before). Perhaps best to just meet and brainstorm, to figure out what we really want to do here?